src/tools/ts-generator/main.cpp

```diff
@@ -44,23 +39,23 @@ std::vector<Data::ThermalCluster*> getClustersToGen(Data::AreaList& areas,
                                                     const std::string& clustersToGen)
 {
     std::vector<Data::ThermalCluster*> clusters;
-    const auto ids = splitStringIntoPairs(clustersToGen, ';', '.');
+    const auto pairsAreaCluster = splitStringIntoPairs(clustersToGen, ';', '.');
 
-    for (const auto& [areaID, clusterID]: ids)
+    for (const auto& [areaName, clusterName]: pairsAreaCluster)
     {
-        logs.info() << "Searching for area: " << areaID << " and cluster: " << clusterID;
+        logs.info() << "Searching for area: " << areaName << " and cluster: " << clusterName;
 
-        auto* area = areas.find(areaID);
+        auto* area = areas.find(areaName);
```

Comment:
`areas.find()` and `thermal.list.findInAll()` takes IDs, which is not the same as names (some char are prohibited in IDs). That's why the list given to the exe is list of IDs.

Revert to old name to explicit the difference